### PR TITLE
Release v5.7.0-rc1

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: doc/release-notes.md [hunk 1]
```diff
@@ -20,7 +20,31 @@ Previously, parameters were stored by default in these locations:
...
+the transactions in the mempool, while `zcash.mempool.size.weighted` is a
+weight-bucketed version of the `zcash.mempool.size.bytes` metric.
+
+The [ZIP 317 weight ratio][weight_ratio] of a transaction is used to bucket its
+logical actions and byte size. A weight ratio of at least 1 means that the
+transaction's fee is at least the ZIP 317 conventional fee, and all of its
+logical actions are considered "paid". A weight ratio lower than 1 corresponds
+to the fraction of the transaction's logical actions that are "unpaid", and
+subject to the unpaid action limit that miners configure for their blocks with
+`-blockunpaidactionlimit`.
```
Comment (daira) [inline doc/release-notes.md:47]:
```suggestion
logical actions are considered "paid". A weight ratio lower than 1 corresponds
to the fraction of the transaction's logical actions that are "paid". The
remaining fraction (i.e. 1 - weight ratio) are subject to the unpaid action
limit that miners configure for their blocks with `-blockunpaidactionlimit`.
```

File: doc/release-notes/release-notes-5.7.0-rc1.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+the transactions in the mempool, while `zcash.mempool.size.weighted` is a
+weight-bucketed version of the `zcash.mempool.size.bytes` metric.
+
+The [ZIP 317 weight ratio][weight_ratio] of a transaction is used to bucket its
+logical actions and byte size. A weight ratio of at least 1 means that the
+transaction's fee is at least the ZIP 317 conventional fee, and all of its
+logical actions are considered "paid". A weight ratio lower than 1 corresponds
+to the fraction of the transaction's logical actions that are "unpaid", and
+subject to the unpaid action limit that miners configure for their blocks with
+`-blockunpaidactionlimit`.
```
Comment (daira) [inline doc/release-notes/release-notes-5.7.0-rc1.md:44]:
```suggestion
logical actions are considered "paid". A weight ratio lower than 1 corresponds
to the fraction of the transaction's logical actions that are "paid". The
remaining fraction (i.e. 1 - weight ratio) are subject to the unpaid action
limit that miners configure for their blocks with `-blockunpaidactionlimit`.
```

**Review by daira — APPROVED** [review]
utACK modulo the incorrect description of weight ratio in the release notes.

**Review by nuttycom — APPROVED** [review]
utACK modulo @daira's suggestions.

File: doc/release-notes.md (hunk 1 above)
Comment (nuttycom) [reply doc/release-notes.md:47]: Fixed in https://github.com/zcash/zcash/pull/6769

File: doc/release-notes/release-notes-5.7.0-rc1.md (hunk 2 above)
Comment (nuttycom) [reply doc/release-notes/release-notes-5.7.0-rc1.md:44]: Fixed in https://github.com/zcash/zcash/pull/6769

**Pull request merged**